src/seedu/addressbook/data/person/DateOfBirth.java

```diff
@@ -0,0 +1,82 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+
+public class DateOfBirth {
+
+    public static final String EXAMPLE = "1.2.1993";
+    public static final String MESSAGE_DOB_CONSTRAINTS = "Numbers in date of birth should be in between periods and should be valid day, month and year.";
```

Comment:
Try not to have a very long (horizontally) line. You might want to use line breaks and `+` in order to keep each line short enough.